src/components/gataca-qr/gataca-qr.tsx

```diff
@@ -385,19 +384,25 @@ export class GatacaQR {
   }
 
   getLink(): string {
-    if (this.v2) {
+    if (this.v !== "3") {
       return this.authenticationRequest;
+    } else {
+      let op = FUNCTION_ROLES[this.qrRole];
```

Comment:
comment [nitpick]: I don't think this works. You can remove everything here from the v1.
Just 


```
const authRequestEncoded = "&dl=" + base64UrlEncode(this.authenticationRequest);
        
return DEEP_LINK_PREFIX + authRequestEncoded;
```


is ok